styles/freedesktop/catppuccin.user.css

```diff
@@ -0,0 +1,155 @@
+==/UserStyle== */
+
+@-moz-document domain('www.freedesktop.org'), domain('specifications.freedesktop.org') {
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  :root[data-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    body {
+        background-color: @base;
+        color: @text;
+    }
+    
+    #content h1,
+    #content h2,
+    #content h3 {
+        color: @accent-color;
+    }
+    
+    .pageheader {
+        background-color: @mantle;
+        border-color: @crust;
+        color: @subtext1;
+    }
+    
+    .pageheader .actions ul li a, input[type="submit"] {
+        background-color: transparent;
+        color: @accent-color;
+        border-color: @accent-color;
+    }
+    
+    input#searchbox {
+        color: @subtext1;
+        background-color: @overlay0;
+        border-color: @overlay1;
+        &:focus {
+           border-color: @accent-color;
+           outline-color: none;
+        }
+    }
+    
+    
+    .headerpath .title {
+        color: @subtext1;
```

Comment:
```suggestion
        color: @text;
```

Reply:
Why? It's not part of the normal text and it isn't colored like the normal text in the original style.

Reply:
On the main page, it's the "www" below the logo, which I wouldn't cassify as title, but also not as normal text. I didn't go the entire website, to find out, where else the element has been used.